### DISCUSSION TITLE
Proxying Docker-Proxy

### DISCUSSION THREAD

**Discussion by Carelvd** [nginx-proxy/nginx-proxy#2284] [Q&A]
I have scanned through some of the previous issues but can't seem to find any commentary on the following setup. Is it possible to proxy Nginx-Proxy via the host ? 

Given a machine which serves ones main site, domain.tld, is it possible to also serve sub domains via an Nginx-Proxy container. I am hoping to acheieve something like the following diagram.

![proxy-3](https://user-images.githubusercontent.com/322018/34615303-1e372b70-f23d-11e7-91af-760c53540554.png)

Would anyone here know if this is possible ? or indeed how to set this up ?

From the [NginX](https://www.nginx.com/blog/ip-transparency-direct-server-return-nginx-plus-transparent-proxy/) it appears that one may use IP transparency. To do so one must route all the traffic from the container back via the hosts Nginx. In my attempts to set this up the traffic seems to return through the default gateway of the containers' network and not through the exposed ports of the container and I'm not sure how one might capture the packets and redirect them through the hosts nginx.

I have asked this on stack overflow/server fault but the question got me a tumbleweed award so that's discouraging.

**Comment by antoniozh** [comment]
I did something like this and just filled the X-Proxy headers in my main nginx instance to feed the nginx-proxy container and routed the traffic via proxy_pass.

**Reply by wnunezc** [reply]
Do you have a basic example? I have some problem with friendly url and my setup is similar.

I made this post, because I cannot evaluate if it is my configuration error or a bug on the side of the image implementation of the proxy in docker.

https://github.com/nginx-proxy/nginx-proxy/issues/2290
